src/Sylius/Bundle/ShopBundle/Resources/config/routing/account/order.yml

```diff
@@ -0,0 +1,11 @@
+# This file is a part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+sylius_shop_account_order:
+    resource: |
+        alias: sylius.order
+        section: shop_account
+        templates: SyliusShopBundle:Account/Order
+        except: ['create', 'update', 'delete']
+        grid: sylius_shop_account_order
+    type: sylius.resource
```

Comment:
I do not think it makes sense to generate the routing here. Let's define the routes manually - will be easier for devs to understand how to customize these in shop. (which they will do much more often than for admin stuff)